### PR TITLE
fix: Integrate task with Jira & GitHub in the demo

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/6349

### CODE REVIEW

File: packages/client/modules/demo/ClientGraphQLServer.ts [hunk 1]
```diff
@@ -248,7 +248,7 @@ class ClientGraphQLServer extends (EventEmitter as GQLDemoEmitter) {
         }
       }
     },
-    TaskFooterIntegrateMenuRootQuery: (_teamId: unknown, userId: string) => {
+    TaskFooterIntegrateMenuQuery: (_teamId: unknown, userId: string) => {
```
Comment (nickoferrall) [inline packages/client/modules/demo/ClientGraphQLServer.ts:251]: These queries were recently renamed. Trying to integrate a task card with an integration or ending a demo causes a crash

File: packages/client/components/TaskFooterIntegrateMenu.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const isIntegrated = (integrations: Integrations) => {
   const {atlassian, github, jiraServer} = integrations
   const hasAtlassian = atlassian?.isActive ?? false
   const hasGitHub = github?.isActive ?? false
-  const hasJiraServer = jiraServer.auth?.isActive ?? false
+  const hasJiraServer = jiraServer?.auth?.isActive ?? false
```
Comment (nickoferrall) [inline packages/client/components/TaskFooterIntegrateMenu.tsx:36]: JiraServer hasn't been added to the demo so we need to check if it exists here. I've created an issue to add GitLab to the demo. It'd be great to create one for Jira Server too if there isn't one already

File: packages/client/modules/demo/ClientGraphQLServer.ts (hunk 1 above)
Comment (JimmyLv) [reply packages/client/modules/demo/ClientGraphQLServer.ts:251]: Just curious, `ops` here refers to simulated operations? The query result here is actually mocked, so it needs to be consistent with the query statement returned by the real GraphQL API. Is my understanding correct? 😁

**Review by JimmyLv — APPROVED** [review]
LGTM!

**Review by JimmyLv — CHANGES_REQUESTED** [review]
Oh, if my [understanding](https://github.com/ParabolInc/parabol/pull/6460/files#r863846573) is correct, I think all of these mocked `RootQuery` need to be renamed, because recently I did the refactoring https://github.com/ParabolInc/parabol/issues/5297 but I didn't realize that I had to take care of these ops in the demo.

```ts
TaskFooterIntegrateMenuRootQuery
NewMeetingSummaryRootQuery
TaskFooterUserAssigneeMenuRootQuery
SuggestMentionableUsersRootQuery
```

**Comment by nickoferrall** [comment]
Sure thing - I've updated two more queries 👍

**Comment by mattkrick** [comment]
tested jira & github, seems to work!

**Pull request merged**